.env.sample

```
@@ -1,20 +1,20 @@
-SENTRY_DSN=''
+VITE_SENTRY_DSN=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)